contracts/modules/capital/SwapOperator.sol

```diff
@@ -509,13 +509,24 @@ contract SwapOperator {
     // Validate there's no current cow swap order going on
     require(!orderInProgress(), "SwapOp: an order is already in place");
 
+    IPool pool = _pool();
+
+    if (assetAddress == ETH) {
```

Comment:
figured we add this here as part of the ugprade so we're prepared for everything

Reply:
Addition looks good to me. Would consider adding a method to unconditionally close an order (say we call it `forceCloseOrder`) for the scenario when `cowSettlement.filledAmount(...)`, `cowSettlement.setPreSignature(...)` or other functions that are called in `closeOrder` are reverting. The only conditions I would leave in place in `forceCloseOrder` would be `validateUID(...)` and `order.validTo` check. Could totally be tackled in a separate issue.